lib/stream/feed.rb

```diff
@@ -38,41 +36,44 @@ def valid_user_id(user_id)
       !user_id.to_s[/^[\w-]+$/].nil?
     end
 
-    def get(params = {})
-      uri = "/feed/#{@feed_url}/"
+    def get(enrich: false, reactions: nil, **params)
```

Comment:
@mircea-cosbuc ruby's support for kw arguments was changed a couple times, are you sure this works on all versions we support?